limit_defaults.go

```diff
@@ -254,7 +254,7 @@ func (cfg *LimitConfig) Apply(c LimitConfig) {
 }
 
 // Scale scales up a limit configuration.
-// memory is the amount of memory that the stack is allowed to consume,
+// memory is the amount of memory in bytes that the stack is allowed to consume,
```

Comment:
I don't think this needs to be specified. Memory is always counted in bytes (in Go).